code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
So I tried that but ended up with funky build issues. Specifically, when I add

    import ColorPicker from 'react-color';

It will load node_modules/react-color/lib/index.js, which includes the line:

    var React = require('react');

Then when I load the dashboard, React complains about being loaded twice. I noticed that in the apps code, we never bother to import React, but browserify doesn't seem to know that. So when react-color imports react, it adds it to the apps bundle.

This is the work around that I figured out, but I don't like it :).

I guess my question is, why isn't React imported explicitly in all the jsx files?